.github/workflows/migration-helper.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+case "${BRANCH_NAME}" in
+"production")
+  export WORKING_STAGE="production"
+  ;;
+*)
+  export WORKING_STAGE="staging"
+  ;;
+esac
+
+echo "SOURCE -> BRANCH_NAME: ${BRANCH_NAME}"
+echo "SOURCE -> WORKING_STAGE: ${WORKING_STAGE}"
+
+#
+# Export environment variables that Hasura CLI will use
+#
+function export_hasura_env_vars() {
+    # Use jq to get Hasura endpoint and admin secret from ZAPPA_SETTINGS
+    export HASURA_GRAPHQL_ENDPOINT=$(echo ${ZAPPA_SETTINGS} | \
+    jq -r ".\"${WORKING_STAGE}\".aws_environment_variables.HASURA_ENDPOINT")
+    
+    export HASURA_GRAPHQL_ADMIN_SECRET=$(echo ${ZAPPA_SETTINGS} | \
+    jq -r ".\"${WORKING_STAGE}\".aws_environment_variables.HASURA_ADMIN_SECRET")
+    
+    # Remove /v1/graphql from the end of the graphql endpoint for Hasura CLI commands
+    export HASURA_GRAPHQL_ENDPOINT=$(basename "${HASURA_ENDPOINT%/v1/graphql}")
```

Comment:
Is this gross? I was hesitant to add yet another secret into `ZAPPA_SETTINGS` or GitHub Actions secrets. The problem comes down to Hasura CLI not needing the full graphql endpoint since it doesn't interface with GraphQL. I could see this catching us off guard if we ever upgraded to a v2 endpoint or something.

Reply:
I don't dislike this, but I can see your point about it maybe catching us off guard at some point. I do agree with you about not wanting to add another secret to the zappa settings. I'm fine with this!

Reply:
This doesn't really bother me!

Reply:
+1, I think this good. Great question though.